tls/s2n_server_hello.c

```diff
@@ -218,7 +218,7 @@ static int s2n_server_hello_parse(struct s2n_connection *conn)
             POSIX_BAIL(S2N_ERR_PROTOCOL_VERSION_UNSUPPORTED);
         }
 
-        uint8_t actual_protocol_version = MIN(conn->server_protocol_version, conn->client_protocol_version);
+        conn->actual_protocol_version = MIN(conn->server_protocol_version, conn->client_protocol_version);
```

Comment:
Functionally, nothing. We were setting a temporary variable before because we needed to compare to actual_protocol_version. Now we need to compare to resume_protocol_version so there's no need for a temporary variable.